<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBinaryDataView_PropertiesEditor</class>
 <widget class="QWidget" name="QBinaryDataView_PropertiesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>133</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Group width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="groupWidth">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="adressBar">
       <property name="text">
        <string>Address bar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="presentationBar">
       <property name="text">
        <string>Presentation bar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="editable">
       <property name="text">
        <string>Editable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
